app/views/distributions/_distribution.html.erb

```diff
@@ -4,15 +4,19 @@
       <% if @distro_type == "leap" %>
         <%= leap_image_tag alt: "Leap Logo", class: "animated jackInTheBox distro-logo" %>
       <% elsif @distro_type == "kubic" %>
-        <%= image_tag 'distributions/kubic-white.svg', alt: "Kubic Logo", class: "animated tada distro-logo" %>
+        <%= image_tag 'distributions/kubic-black.svg', alt: "Kubic Logo", class: "animated tada distro-logo" %>
+      <% elsif @distro_type == "microos" %>
+        <%= image_tag 'distributions/microos-white.svg', alt: "MicroOS Logo", class: "animated tada distro-logo" %>
       <% else %>
-        <%= image_tag 'distributions/tumbleweed-white.svg', alt: "Tumbleweed Logo", class: "animated zoomInDown distro-logo" %>
+        <%= image_tag 'distributions/tumbleweed-black.svg', alt: "Tumbleweed Logo", class: "animated zoomInDown distro-logo" %>
       <% end %>
-      <h1 class="display-3 text-white">
+      <h1 class="display-3 text-<%= @text %>">
```

Comment:
nitpick: I feel like `@text` is a bit misleading for a name, without looking at the diff or the controller I would not know that `@text` is the text colour. I'd call it `@text_colour` instead.

Reply:
I was planning to do @bg_colour and @fg_colour, so we can use that with assets as well